stories/object-status/object-status.stories.js

```diff
@@ -118,43 +127,41 @@ export const inverted = () => `
 <br><br>
 
 <span class="fd-object-status fd-object-status--inverted">Inverted Neutral</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--negative sap-icon--status-negative"></span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--negative sap-icon--status-negative">Negative</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--critical sap-icon--status-critical">Critical</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--positive sap-icon--status-positive">Positive</span>
-<span class="fd-object-status fd-object-status--inverted fd-object-status--informative sap-icon--hint">Informative</span>
-<span class="fd-object-status fd-object-status--inverted sap-icon--to-be-reviewed">Neutral</span>
+<span class="fd-object-status fd-object-status--inverted fd-object-status--negative"><i class="fd-object-status__icon sap-icon--status-negative" aria-label="Inverted Negative Icon"></i></span>
```

Comment:
This story still needs to be fixed to remove right-padding when there is no text:
![image](https://user-images.githubusercontent.com/43764832/93153000-0c0e5200-f6b5-11ea-9bea-9fbc2b69056e.png)